<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Volker Krause &lt;vkrause@kde.org&gt;, KovoKs &lt;tomalbers@kde.nl&gt;</author>
 <class>SMTPSettings</class>
 <widget class="QWidget" name="SMTPSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>438</height>
   </rect>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <widget class="KTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="smptTab">
      <attribute name="title">
       <string comment="general smtp settings">General</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="hostLabel">
         <property name="text">
          <string>Outgoing mail &amp;server:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>kcfg_host</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="KLineEdit" name="kcfg_host"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="portLabel">
         <property name="text">
          <string>&amp;Port:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>kcfg_port</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout">
         <item>
          <widget class="KIntNumInput" name="kcfg_port">
           <property name="value">
            <number>25</number>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>231</width>
             <height>23</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="KButtonGroup" name="kcfg_encryption">
         <property name="title">
          <string>Encryption</string>
         </property>
         <layout class="QVBoxLayout">
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QRadioButton" name="none">
              <property name="text">
               <string>&amp;None</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="ssl">
              <property name="text">
               <string>&amp;SSL</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="tls">
              <property name="text">
               <string>&amp;TLS</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="KPushButton" name="checkCapabilities">
            <property name="text">
             <string>Check &amp;What the Server Supports</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="checkCapabilitiesProgress">
            <property name="visible">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="advancedTab">
      <attribute name="title">
       <string comment="advanced smtp settings">Advanced</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0" colspan="3">
        <widget class="QLabel" name="noAuthPossible">
         <property name="visible">
          <bool>false</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="text">
          <string>This server does not support authentication</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="usernameLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>&amp;Login:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>kcfg_userName</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="KLineEdit" name="kcfg_userName">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="passwordLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>P&amp;assword:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>password</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="KLineEdit" name="password">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="whatsThis">
          <string>The password to send to the server for authorization.</string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QCheckBox" name="kcfg_storePassword">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>&amp;Store SMTP password</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="3">
        <widget class="KButtonGroup" name="kcfg_authenticationType">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="title">
          <string>Authentication Method</string>
         </property>
         <layout class="QGridLayout">
          <item row="0" column="0">
           <layout class="QGridLayout">
            <item row="0" column="0">
             <widget class="QRadioButton" name="login">
              <property name="text">
               <string>&amp;LOGIN</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QRadioButton" name="plain">
              <property name="text">
               <string>&amp;PLAIN</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="crammd5">
              <property name="text">
               <string>CRAM-MD&amp;5</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QRadioButton" name="digestmd5">
              <property name="text">
               <string>&amp;DIGEST-MD5</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QRadioButton" name="gssapi">
              <property name="text">
               <string>&amp;GSSAPI</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QRadioButton" name="ntlm">
              <property name="text">
               <string>&amp;NTLM</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="6" column="0" colspan="3">
        <widget class="KSeparator" name="kseparator_3"/>
       </item>
       <item row="7" column="0" colspan="3">
        <widget class="QCheckBox" name="kcfg_specifyHostname">
         <property name="text">
          <string>Sen&amp;d custom hostname to server</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="QLabel" name="hostnameLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Hos&amp;tname:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>kcfg_localHostname</cstring>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="KLineEdit" name="kcfg_localHostname">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="3">
        <widget class="KSeparator" name="kseparator_2"/>
       </item>
       <item row="10" column="0" colspan="2">
        <widget class="QLabel" name="precommandLabel">
         <property name="text">
          <string>Precommand:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="10" column="2">
        <widget class="KLineEdit" name="kcfg_precommand"/>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QCheckBox" name="kcfg_requiresAuthentication">
         <property name="text">
          <string>Server &amp;requires authentication</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KButtonGroup</class>
   <extends>QGroupBox</extends>
   <header>kbuttongroup.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KSeparator</class>
   <extends>QFrame</extends>
   <header>kseparator.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>kcfg_host</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>kcfg_requiresAuthentication</sender>
   <signal>toggled(bool)</signal>
   <receiver>usernameLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_requiresAuthentication</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_userName</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_requiresAuthentication</sender>
   <signal>toggled(bool)</signal>
   <receiver>passwordLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>61</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_requiresAuthentication</sender>
   <signal>toggled(bool)</signal>
   <receiver>password</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_requiresAuthentication</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_storePassword</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>62</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_requiresAuthentication</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_authenticationType</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_specifyHostname</sender>
   <signal>toggled(bool)</signal>
   <receiver>hostnameLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>334</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_specifyHostname</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_localHostname</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>327</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
